src/extensions/mountainview/MVSortingView/MVSortingView.tsx

```diff
@@ -1,6 +1,8 @@
-import { faSquare } from '@fortawesome/free-regular-svg-icons'
-import { faEnvelope, faGrinHearts, faPencilAlt, faSocks, faVials } from '@fortawesome/free-solid-svg-icons'
+import { faPencilAlt } from '@fortawesome/free-solid-svg-icons'
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { GraphicEq, Settings, Visibility } from '@material-ui/icons'
```

Comment:
Mostly switching from FontAwesome icons to material icons. Many font awesome icons require license fee.